DNN Platform/Library/Startup.cs

```diff
@@ -2,26 +2,31 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
 
-namespace DotNetNuke
-{
-    using DotNetNuke.Abstractions;
-    using DotNetNuke.Common;
-    using DotNetNuke.DependencyInjection;
-    using DotNetNuke.Entities.Portals;
-    using DotNetNuke.UI.Modules;
-    using DotNetNuke.UI.Modules.Html5;
-    using Microsoft.Extensions.DependencyInjection;
+namespace DotNetNuke
+{
+    using DotNetNuke.Abstractions;
+    using DotNetNuke.Common;
+    using DotNetNuke.DependencyInjection;
+    using DotNetNuke.Entities.Controllers;
+    using DotNetNuke.Entities.Portals;
+    using DotNetNuke.UI.Modules;
+    using DotNetNuke.UI.Modules.Html5;
 
-    public class Startup : IDnnStartup
-    {
-        public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddSingleton<WebFormsModuleControlFactory>();
-            services.AddSingleton<Html5ModuleControlFactory>();
-            services.AddSingleton<ReflectedModuleControlFactory>();
-
-            services.AddTransient(x => PortalController.Instance);
-            services.AddTransient<INavigationManager, NavigationManager>();
-        }
-    }
-}
+    using Microsoft.Extensions.DependencyInjection;
+
+    using INewHostController = DotNetNuke.Abstractions.Entities.Controllers.IHostController;

```

Comment:
I understand WHY this is done, but this smells really funny

Reply:
See comment above - https://github.com/dnnsoftware/Dnn.Platform/pull/3990#discussion_r471475756. Let's have the discussion in my alias statement there to prevent thread splitting

Reply:
We have renamed the interface to `IHostSettingsService` and there is no longer any need for the using alias. 

https://github.com/dnnsoftware/Dnn.Platform/pull/3990#discussion_r474019228